src/main/java/sg/ncl/ConnectionProperties.java

```diff
@@ -331,6 +331,10 @@ public String saveImage() {
         return HTTP_MODE + sioAddress + ":" + sioPort + "/" + imageEndpoint;
     }
 
+    public String deleteImage(String teamId, String imageName) {
+        return HTTP_MODE + sioAddress + ":" + sioPort + "/" + imageEndpoint + "/" + imageName + "/delete" + "?teamId=" + teamId ;
```

Comment:
Modify endpoint to deleteImage(String imageId)
- /imageEndpoint/imageId
Should not be using /imageEndpoint/imageName/delete?teamId=teamId